components/avatars.css

```diff
@@ -0,0 +1,67 @@
+@import url(/stylesheets/style.css);
+@import url(/components/button.css);
+
+.avatar > img {
+  border: solid var(--primary-100);
+  height: auto;
+  width: 7rem;
+  border-radius: 50%;
+  text-align: center;
```

Comment:
No need for text-align on images.

Reply:
I have made the required changes. Please approve them.